packages/smooth_app/lib/widgets/selector_screen/smooth_screen_selector_provider.dart

```diff
@@ -25,6 +25,8 @@ abstract class PreferencesSelectorProvider<T>
   Future<List<T>> onLoadValues();
   T getSelectedValue(List<T> values);
 
+  bool _mounted = true;
```

Comment:
Please use another name: `mounted` has a standard use in `Widget`s and that may cause confusion. E.g. `attached` or `running`.